packages/ui/core-components/src/lib/atoms/inputs/slider/Slider.svelte

```diff
@@ -72,11 +49,61 @@
 	/** @type {string | undefined} */
 	export let fmt = undefined;
 
-	$: $inputs[name] = value;
-
 	/** @type {string} */
 	export let size = '';
 
+	/** @type {string} */
+	let format_object;
+
+	function validateNumber(value, name) {
+		value = toNumber(value);
+		if (isNaN(value)) {
+			console.error(`${name} must be a number`);
+			return undefined;
+		}
+		return value;
+	}
+
+	function checkMinMax(min, max) {
+		if (min > max) {
+			console.error('min cannot be greater than max');
+		}
+	}
+
+	if (min !== undefined) {
+		min = validateNumber(min, 'min');
+		if (min !== undefined && max !== undefined) {
+			checkMinMax(min, max);
+		}
+	}
+
+	if (max !== undefined) {
+		max = validateNumber(max, 'max');
+		if (max !== undefined && min !== undefined) {
+			checkMinMax(min, max);
+		}
+	}
```

Comment:
`checkMinMax` is called twice, it only needs to be called once

```suggestion
	if (min !== undefined) {
		min = validateNumber(min, 'min');
	}
	if (max !== undefined) {
		max = validateNumber(max, 'max');
	}
	if (max !== undefined && min !== undefined) {
		checkMinMax(min, max);
	}
```